va/hci/stage6/openstackcontrolplane.yaml

```diff
@@ -93,19 +93,20 @@ spec:
       route: {}
```

Comment:
The `glance.apiOverride` also needs adjusting here to match the new map

Reply:
Done